apollo-http-cache/src/main/kotlin/com/apollographql/apollo3/cache/http/DiskLruHttpCache.kt

```diff
@@ -104,6 +137,51 @@ class DiskLruHttpCache(private val fileSystem: FileSystem, private val directory
     }
   }
 
+  /**
+   * A [BufferedSource] that writes to the given cache sink as it is read.
+   */
+  private class WriteToCacheSource(
+      private val originalBody: Source,
+      private val cacheSink: Sink,
```

Comment:
Nit: what about proxy? 

```suggestion
   * A [Source] that writes to the given cache sink as it is read.
   */
   private class ProxySource(
      private val originalSource: Source,
      private val sink: Sink,
```

Reply:
I hesitated naming it more "generically" but because of the `cacheEditor` it's pretty specific to the cache - OTOH  `ProxySource` is pretty clear too 👍

Reply:
Your call. The only thing I'm pretty positive about is that it is a `Source` and not a `BufferedSource`